setup.sh

```diff
@@ -215,27 +223,8 @@ function config_generator() {
 function config() {
 
     echo "-------------------------"
-    echo "Generating config file"
+    echo "Please use freqtrade new-config -c config.json to generate a new configuration file."
```

Comment:
`'Please use "freqtrade new-config -c config.json" to generate a new configuration file.'`
-- to have the command in double quotes, like `git`, for instance, prints its suggestions for the commands to be run.
(?)

Reply:
I'll use single quotes for now - so it's aligned to the rest of this script...